app/react/package.json

```diff
@@ -67,6 +67,11 @@
     "react": "^16.8.0 || ^17.0.0",
     "react-dom": "^16.8.0 || ^17.0.0"
   },
+  "peerDependenciesMeta": {
+    "typescript": {
```

Comment:
Shouldn't `typescript` be listed as `peerDependencies` too? Or having this `peerDependenciesMeta` info is enough and Yarn will "do it" for us under the hood?

Reply:
Technically it would be more correct to list it, yes. However when we introduced the feature some maintainers were concerned that it would print more warnings on older package managers (for example for their users running npm 5), so we improved it by making `peerDependenciesMeta` imply a corresponding peer dependency on `*` (this way older package managers just ignore the peer dependency). So in short, it's as you prefer, we support both 🙂